File: Language_Coder/src/loop/Code149.java
package loop;

import java.util.Scanner;

public class Code149 {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		int n = scan.nextInt();
		int num = 1;

		for (int i = 0; i < n; i++) {
			for (int j = 0; j < n; j++) {
				System.out.print(num);
				num += 2;
				num %= 10;
				if (j != n - 1)
					System.out.print(" ");
			}
			System.out.println();
		}
	}

}

File: Language_Coder_Cpp/Code149/Code149.cpp
// Code149.cpp

#include <iostream>
using namespace std;

int main()
{
	int num;
	cin >> num;

	int s = 1;

	for (int i = 0; i < num; i++) {
		for (int j = 0; j < num; j++) {
			cout << s << " ";
			s += 2;
			if (s >= 10) {
				s = 1;
			}
		}
		cout << endl;
	}
}